community/kernel/src/test/java/org/neo4j/kernel/impl/index/schema/fusion/FusionIndexReaderTest.java

```diff
@@ -96,45 +93,6 @@ public void closeIteratorMustCloseNativeAndLucene() throws Exception
         verify( luceneIter, times( 1 ) ).close();
     }
 
-    @Test
```

Comment:
What happened to these tests? Is this no longer true?

Reply:
Because they are now Resources they are not allowed to throw checked exceptions on close.